test/cypress/support/commands.js

```diff
@@ -90,6 +91,10 @@ Cypress.Commands.add('giLogin', (username, {
       }
       await sbp('gi.actions/identity/login', { username, password })
       await sbp('controller/router').push({ path: '/' }).catch(e => {})
+
+      return new Promise((resolve) => {
+        sbp('okTurtles.events/on', LOGIN, resolve)
+      })
```

Comment:
I think this can be removed. And if for some reason it was needed, the promise should be created before `'gi.actions/identity/login'` is called, and it should be created with `/once` instead of `/on`. But I think this can be safely removed. The latest changes in `master` should have fixed recent heisenbugs

Reply:
Yeah. Removed. Nothing failed anyway.